### PR TITLE
Fix so warnings don't occur

### PR DESCRIPTION
**Pull request by marksmall**
I noticed when using this library, a few warnings, especially when running tests. I have now fixed, I think these problems, however I can't prove it here as there is ability to run tests.

### CODE REVIEW

File: dat_utils/__init__.py [hunk 1]
```diff
@@ -1,11 +1,4 @@
-"""
-    .___       __                    __  .__.__          
-  __| _/____ _/  |_           __ ___/  |_|__|  |   ______
- / __ |\__  \\   __\  ______ |  |  \   __\  |  |  /  ___/
-/ /_/ | / __ \|  |   /_____/ |  |  /|  | |  |  |__\___ \ 
-\____ |(____  /__|           |____/ |__| |__|____/____  >
-     \/     \/                                        \/ 
-"""
+""" dat-utils """
```
Comment (R2ZER0) [inline dat_utils/__init__.py]: :cry:

File: dat_utils/__init__.py [hunk 2]
```diff
@@ -1,11 +1,4 @@
...
```
Comment (allynt) [inline dat_utils/__init__.py]: Why'd this go?

**Review by allynt — APPROVED** [review]

File: dat_utils/__init__.py (hunk 1 above)
Comment (marksmall) [reply dat_utils/__init__.py]: yeah I know, if anyone know how to keep it, I'm fine. The reason I removed it was I was getting the following warning:

```bash
.venv/src/django-astrosat-users/astrosat_users/__init__.py:14
  /home/app/.venv/src/django-astrosat-users/astrosat_users/__init__.py:14: DeprecationWarning: invalid escape sequence \ 
    """
```

File: dat_utils/__init__.py (hunk 2 above)
Comment (marksmall) [reply dat_utils/__init__.py]: see response to rikki's comment

File: dat_utils/__init__.py (hunk 1 above)
Comment (R2ZER0) [reply dat_utils/__init__.py]: You can use the `r` prefix to ignore escape sequences, so:
```
r"""
...
"""
```

https://docs.python.org/3.7/reference/lexical_analysis.html#string-and-bytes-literals

Comment (marksmall) [reply dat_utils/__init__.py]: I did that with the others as you can see. I thought this was only for regex's, but happy to use it here, if it works. My problem is, I have no way of proving it in this library, testing has not been setup. I didn't want to go to that level for this, but testing should be added at some point.

I'll add the `r` here as well, but we won't know if it works or not until we merge and use it.

Comment (marksmall) [reply dat_utils/__init__.py]: I've re-introduced the ascii art and prefixed it with `r` to treat it as a **raw string**. We will need to add tests at some point to prove we are not introducing any such problems in the future, but that can be done later.

**Pull request merged**